### PR TITLE
Add google analytics to the site

### PR DESCRIPTION
**Pull request by titlescreen**
Also update the cookies page with Google info

### CODE REVIEW

File: app/views/static_pages/cookies.html.erb [hunk 1]
```diff
@@ -67,21 +67,21 @@
...
+          <td class="govuk-table__cell">_ga</td>
           <td class="govuk-table__cell">This helps us count how many people visit <%= t("service_name") %> by tracking if you’ve visited before</td>
           <td class="govuk-table__cell">After 2 years</td>
         </tr>
         <tr>
           <tr class="govuk-table__row">
-          <td class="govuk-table__cell">XXXX_gid</td>
+          <td class="govuk-table__cell">_gid</td>
           <td class="govuk-table__cell">This helps us count how many people visit <%= t("service_name") %> by tracking if you’ve visited before</td>
           <td class="govuk-table__cell">After 24 hours</td>
```
Comment (erbridge) [inline app/views/static_pages/cookies.html.erb]: Are we confident that these are the cookies used by the tag manager, too?

**Comment by erbridge** [comment]
This needs to set up the infrastructure side of things to pass the env var in before we'll be able to turn it on.

File: app/views/static_pages/cookies.html.erb (hunk 1 above)
Comment (titlescreen) [reply app/views/static_pages/cookies.html.erb]: When we were looking at implementing Tag Manager it gave people too much opportunity to be naughty so we changed it to standard Google Analytics. It is using the Google Analytics Global Tag, not Google Tag Manager.

Comment (erbridge) [reply app/views/static_pages/cookies.html.erb]: Ah ok. It looks to me like we're still including the tag manager script in the header, though. Or am I mistaken?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -18,6 +18,12 @@
 
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 
+    <% if ENV["GOOGLE_ANALYTICS_ID"] %>
+
+      <%= javascript_include_tag "https://www.googletagmanager.com/gtag/js?id=#{ENV["GOOGLE_ANALYTICS_ID"]}" %>
```
Comment (erbridge) [inline app/views/layouts/application.html.erb]: Does this line belong here if we're not using Tag Manager?

Comment (tekin) [reply app/views/layouts/application.html.erb]: It's what Google say to "copy and paste" into the site to enable Google Analytics, along with the code below.

Comment (erbridge) [reply app/views/layouts/application.html.erb]: Ah ok. Misled by the name.

Comment (tekin) [reply app/views/layouts/application.html.erb]: Yeah, it's confusing, but it looks like even without using full Tag Manager, Google Analytics is still loaded using this JS.

File: azure/resource_groups/app/parameters/development.template.json [hunk 3]
```diff
@@ -139,6 +139,14 @@
         "secretName": "DfeSignInApiEndpoint"
       }
     },
+    "GOOGLE_ANALYTICS_ID": {
+      "reference": {
+        "keyVault": {
+          "id": "${keyVaultId}"
+        },
+        "secretName": "GoogleAnalyticsId"
```
Comment (robbiepaul) [inline azure/resource_groups/app/parameters/development.template.json]: We could argue that this doesn't need to be stored in KeyVault as it's not secret. We could just provide it as a value?

Comment (erbridge) [reply azure/resource_groups/app/parameters/development.template.json]: Putting it in the Key Vault gives us some advantages in that we can change it and redeploy without committing changes. Though it does add some indirection, and increase the number of "secrets" we store that aren't secret.

**Pull request merged**